tests/utils.py

```
@@ -685,3 +688,46 @@ def verify_rwx_default_storage(client: DynamicClient) -> None:
+        namespace=namespace_name,
+        get_all=True,
+    )
+    pod_names = None
+    not_running_pods = None
+    try:
+        current_check = 0
+        for sample in samples:
+            if sample:
+                not_running_pods = get_not_running_pods(pods=sample)
+                pod_names = [pod.name for pod in sample]
+                LOGGER.info(f"All {pod_prefix} pods: {pod_names}, not running: {not_running_pods}")
```

Ruff (0.14.14)

[warning] Logging statement uses f-string

(G004)